sdk/eventgrid/azure-eventgrid/dev_requirements.txt

```diff
@@ -1 +1,8 @@
--e ../../../tools/azure-sdk-tools
\ No newline at end of file
+-e ../../core/azure-core
```

Comment:
Always install the more generic first:
- devtools
- sdk-tools
- core
- identity
- all other packages